### ISSUE TITLE
Make sure styling is consistent across pages

### ISSUE THREAD

**Issue by alexisbarnes** [ncopenpass/CampaignFinance#84]
This also includes covering some ADA compliant things such as:
- The color contrast is good
- Images have alt text
- Not skipping header tags across the page (only use for headings and not things like links)
- Buttons and Search have a hover state
- Links don't open in a new tab (maybe except report error button until we change it away from the google form)

And check with either https://wave.webaim.org or https://developers.google.com/web/tools/lighthouse/

**Comment by SchaeStewart** [comment]
IBM has an accessibility toolkit with a lot of good resources for keeping things accessible: https://www.ibm.com/able/toolkit

**Comment by funkybunch** [comment]
Using the WCAG 2.1 AA Standard for accessibility: https://w3.org/TR/WCAG21/

**Comment by JMMaok** [comment]
Also align values in columns. `Amounts ` should be either right-aligned or decimal-aligned. That helps visually judge orders of magnitude.
